backend/lightning/handlers.go

```diff
@@ -176,6 +176,24 @@ func (lightning *Lightning) PostSendPayment(r *http.Request) interface{} {
 		return responseDto{Success: false, ErrorMessage: err.Error()}
 	}
 
+	invoice, err := breez_sdk.ParseInvoice(jsonBody.Bolt11)
+	if err != nil {
+		return responseDto{Success: false, ErrorMessage: err.Error()}
+	}
+
+	nodeState, err := lightning.sdkService.NodeInfo()
+	if err != nil {
+		return responseDto{Success: false, ErrorMessage: err.Error()}
+	}
+
+	amount := invoice.AmountMsat
+	if jsonBody.AmountMsat != nil {
+		amount = jsonBody.AmountMsat
+	}
+	if *amount > nodeState.ChannelsBalanceMsat {
```

Comment:
can amount be nil? maybe best to check to avoid a panic

Reply:
Should not be possible, but added just in case